.github/CODEOWNERS

```diff
@@ -16,13 +16,19 @@
 
 * @department-of-veterans-affairs/va-platform-cop-frontend
 
+
+# Platform Design System
+
 src/applications/_mock-form @department-of-veterans-affairs/platform-design-system-fe
 src/applications/burial-poc-v6 @department-of-veterans-affairs/platform-design-system-fe
```

Comment:
I realize this was here already but I don't think the design system should be assigned to a specific app like this. I'm not sure who is responsible for `burial-poc-v6` though.

Also `src/applications/_mock-form` is an unknown for me. Maybe better for @department-of-veterans-affairs/platform-va-product-forms?

Reply:
Feel free to make any comments about this file, whether they changed or not. :)

I made an update to re-assign those.